app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str,
```

Comment:
Please read checklist before pushing your solution.
Point 5, move the name to a new line :)